py3status/modules/external_script.py

```diff
@@ -13,6 +13,8 @@
     cache_timeout: how often we refresh this module in seconds
         (default 15)
     format: see placeholders below (default '{output}')
+    ignore_stderr: do not treat output on stderr as failure
+        (defaut True)
```

Comment:
Hi, I'm your Travis today. What is `defaut`? I do not know if it is good idea to leave this on by `defaut`.

Reply:
As I mentioned in the first message, I don't think the current behavior in master makes any sense, very often scripts print some additional information on stderr and it doesn't mean the script failed. Even black formatter prints some emojis on stderr upon successful completion